modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/bulk/BulkRunAction.java

```diff
@@ -162,6 +162,14 @@ public BulkStatus run() throws IOException, OperationException {
         }
         if (queryLimit > 0) {
             builder.queryLimit(queryLimit);
+        } else if (action == "automation-ui" && params.containsKey(OPERATION_ID)) {
```

Comment:
`"automation-ui".equals(action)` and maybe worth putting "automation-ui" in a constant, or even use `AutomationBulkActionUi.ACTION_NAME`?

Reply:
I keep it as string because the definition is done in the configuration